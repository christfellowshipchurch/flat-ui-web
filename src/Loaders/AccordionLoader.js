import React from 'react';

import {
  Container, Row, Col,
} from 'reactstrap';

import '../styles/components/Loaders.scss'

const Accordion = () => {
  return (
    <Container className="my-5">
      <Row>
        {[1, 2, 3, 4, 5, 6].map((n) => (
          <Col xs="12" md="6" key={n}>
            <div className="ph-g ph-accordion m-1"></div>
          </Col>
        ))}
      </Row>
    </Container>
  );
}

const propTypes = {
};
const defaultProps = {
};


Accordion.propTypes = propTypes;
Accordion.defaultProps = defaultProps;

export default Accordion;
