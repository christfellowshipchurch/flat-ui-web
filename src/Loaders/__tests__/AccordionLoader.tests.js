import React from 'react'
import AccordionLoader from '../AccordionLoader'
import renderer from 'react-test-renderer'

describe('AccordionLoader', () => {
    it('renders as a placeholder component', () => {
        const tree = renderer.create(<AccordionLoader />)

        expect(tree).toMatchSnapshot()
    })
})