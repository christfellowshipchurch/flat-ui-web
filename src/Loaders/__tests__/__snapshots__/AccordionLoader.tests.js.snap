// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordionLoader renders as a placeholder component 1`] = `
<div>
  <div>
    <div />
  </div>
  <div>
    <div />
  </div>
  <div>
    <div />
  </div>
  <div>
    <div />
  </div>
  <div>
    <div />
  </div>
  <div>
    <div />
  </div>
</div>
`;
